.azure-pipelines/auto.yml

```diff
@@ -0,0 +1,356 @@
+#
+# Azure Pipelines "auto" branch build for Rust on Linux, macOS, and Windows.
+#
+
+pr: none
+ trigger:
+ - auto
```

Comment:
Could we allow it to trigger on `try` branch to test its behavior before merging?

Reply:
Yep, I'll do a try build when the repo is enabled on the Azure side.